.github/workflows/test-slow-imports-rule.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Install
         run: pnpm install --frozen-lockfile --prefer-offline
       - name: Add slow import
-        run: echo 'import "lodash";' >> src/internal/constants.ts
+        run: echo 'import "lodash";' >> src/internal/core/config/config-resolution.ts
```

Comment:
Let's keep the previous one too. The more files we use, the better.

Reply:
Hmm, but wouldn't the first fail hide the rest of the violations?